public/javascripts/Choropleth.js

```diff
@@ -1,241 +1,327 @@
+                 p > 90 ? params.regionColors[0] :
+                    p > 80 ? params.regionColors[1] :
+                        p > 70 ? params.regionColors[2] :
+                            p > 60 ? params.regionColors[3] :
+                                p > 50 ? params.regionColors[4] :
+                                    p > 40 ? params.regionColors[5] :
+                                        p > 30 ? params.regionColors[6] :
+                                            p > 20 ? params.regionColors[7] :
+                                                p > 10 ? params.regionColors[8] :
+                                                    params.regionColors[9];
+    }
 
-        // Logs when a region is selected from the choropleth and 'Click here' is clicked
-        // Logs are of the form "Click_module=Choropleth_regionId=<regionId>_distanceLeft=<"0", "<1", "1" or ">1">_target=audit"
-        // Log is stored in WebpageActivityTable
-        $("#choropleth").on('click', '.region-selection-trigger', function () {
-            var regionId = $(this).attr('regionId');
-            var ratesEl = rates.find(function(x){
-                return regionId == x.region_id;
-            });
-            var compRate = Math.round(100.0 * ratesEl.rate);
-            var milesLeft = Math.round(0.000621371 * (ratesEl.total_distance_m - ratesEl.completed_distance_m));
-            var distanceLeft = "";
-            if (compRate === 100){
-                distanceLeft = "0";
-            }
-            else if (milesLeft === 0){
-                distanceLeft = "<1";
-            }
-            else if (milesLeft === 1){
-                distanceLeft = "1";
-            }
-            else{
-                distanceLeft = ">1";
+    /**
+     * This function finds the color for a specific region of the accessibility choropleth.
+     * 
+     * @param {*} rate Object from which information about labels is retrieved.
+     */
+    function findAccessibilityChoroplethColor(rate) {
+        var totalIssues = 0;
+        for (var issue in rate.labels) {
+            if (rate.labels.hasOwnProperty(issue)) {
+                totalIssues += rate.labels[issue];
             }
+        }
+        var significantData = rate.rate >= .3;
+        var fillColor = significantData ? getColor(1000.0 * totalIssues / rate.completed_distance_m) : '#888';
+        var fillOpacity = significantData ? 0.4 + (totalIssues / rate.completed_distance_m) : .25;
+        return {
+            color: '#888',
+            weight: 1,
+            opacity: 0.25,
+            fillColor: fillColor,
+            fillOpacity: fillOpacity
+        }
+    }
 
-            var data = "Click_module=Choropleth_regionId="+regionId+"_distanceLeft="+distanceLeft+"_target=audit";
-            postToWebpageActivity(data);
-        });
+    /**
+     * This function sets the specific popup content of each region of the accessibility choropleth.
```

Comment:
This comment feels out of date. Something more like "Gets issue count HTML to add to popups on the results page."

Then also rename the function. I don't think "AcessibilityChoropleth" was every a good or informative name.